document/core/syntax/instructions.rst

```diff
@@ -71,6 +93,49 @@ These operations closely match respective operations available in hardware.
      \K{f}\X{nn}\K{.}\CONVERT\K{\_i}\X{mm}\K{\_}\sx \\&&|&
      \K{i}\X{nn}\K{.}\REINTERPRET\K{\_f}\X{nn} ~|~
      \K{f}\X{nn}\K{.}\REINTERPRET\K{\_i}\X{nn} \\&&|&
+     \K{v8x16.}\SHUFFLE ~|~ \K{v8x16.}\SWIZZLE \\&&|&
+     \X{vxx}\K{.}\SPLAT \\&&|&
+     \K{i8x16.}\EXTRACTLANE\K{\_}\sx ~|~
+     \K{i16x8.}\EXTRACTLANE\K{\_}\sx \\&&|&
+     \K{i32x4.}\EXTRACTLANE ~|~
+     \K{i64x2.}\EXTRACTLANE \\&&|&
+     \X{fxx}\K{.}\EXTRACTLANE \\&&|&
+     \X{vxx}\K{.}\REPLACELANE \\&&|&
+     \X{ixx}\K{.}\irelop \\&&|&
+     \X{fxx}\K{.}\frelop \\&&|&
+     \K{i8x16.}\viunop ~|~
+     \K{i16x8.}\viunop ~|~
+     \K{i32x4.}\viunop \\&&|&
+     \K{i64x2.}\NEG \\&&|&
```

Comment:
I wonder if it would be clearer to use patterns everywhere and exclude forms that are not allowed; there doesn't seem to be precedent for that, however.

Reply:
That will be kinda neat, but the asymmetry is pretty bad, such an exclusion list might end up being unwieldy too. Probably a short form for i8x16, i16x8, i32x4 will get us part way there (what would a name for that be small_ixx?)

Reply:
Yeah, not sure the best name to use. Thoughts on this, @rossberg?

Reply:
In the context of memory access I've called the smaller int types "packed", but does that really fit here? Also, it looks like the irregularity is high enough that we need at least two subsets. And then multiple sets of viunop/vibinop. It may not be worth it.

Reply:
Yea, packed doesn't fit as well here. We kinda use "pack" to mean packing integers/floats side by side. I agree that it might not be worth it. Let's keep this for now, as I work on this, I'll stare at this and see if I can come up with something better. (And I think future work on the validation/execution portion will help us see if there is a better way to organize this.)